src/librustc_mir/borrow_check/nll/region_infer/error_reporting/mod.rs

```diff
@@ -54,6 +54,31 @@ enum Trace {
     NotVisited,
 }
 
+/// Various pieces of state used when reporting borrow checker errors.
+pub struct ErrorReportingCtx<'a, 'b, 'tcx> {
+    #[allow(dead_code)] // FIXME(mark-i-m): used by outlives suggestions
+    rinfcx: &'b RegionInferenceContext<'tcx>,
```

Comment:
```suggestion
    region_infcx: &'b RegionInferenceContext<'tcx>,
```

Reply:
I believe the current naming is consistent with everywhere else, but I can change it if you feel strongly.

Reply:
I'd say the current scheme isn't very readable. :)

Reply:
Changed it. Actually, it turns out that due to this refactoring it was pretty painless.